### ISSUE TITLE
"Could not find filebucket  specified in backup" error on 3.7.4

### ISSUE THREAD

**Issue by centromere** [claranet/puppet-consul_template#9]
The following occurs on Puppet 3.7.4 when I try to use this module:

```
root@lb01:/etc/puppet# mkdir -p inhouse/mymodule/templates
root@lb01:/etc/puppet# touch inhouse/mymodule/templates/test.ctmpl.erb
root@lb01:/etc/puppet# puppet apply
include consul_template
consul_template::watch { 'common':
    template    => 'mymodule/test.ctmpl.erb',
    destination => '/tmp/test.json',
    command     => 'true',
}
Notice: Compiled catalog for lb01.local in environment production in 2.61 seconds
Error: Could not find filebucket  specified in backup
```

**Comment by gdhbashton** [comment]
That's kind of funky.

This is coming from the `concat` module which sets the `backup` to a default value of `puppet`. This is passed straight through to Puppet's native `file()` resource:

https://docs.puppetlabs.com/references/latest/type.html#file-attribute-backup

It looks like in naked `puppet apply` mode (which is what I always use anyway...) the `puppet` local filebucket is not being created. I'll try and dig a little more and please update if you find anything more in the meantime!

Cheers,
Gavin.

**Comment by centromere** [comment]
I did some digging and found this:

https://tickets.puppetlabs.com/browse/MODULES-1933
https://github.com/puppetlabs/puppetlabs-concat/commit/253f4975c72aaa812579576da369f57194074c08

You're right that the bug is located in concat, and it appears that the fix was committed to the 1.2.x branch, but no release has been made yet. Consequently, I am closing this ticket.
